backend/app/adapter/graphql/schema.go

```diff
@@ -16,10 +16,24 @@ type Mutation {
 
 type AuthQuery {
 	URL(alias: String!, expireAfter: Time): URL
+	changeLog: ChangeLog!
+}
+
+type ChangeLog {
+  	changes: [Change!]!
+  	lastViewedAt: Time
+}
+
+type Change {
+  	id: String!
+  	title: String!
+  	summaryMarkdown: String
+  	releasedAt: Time
```

Comment:
```suggestion
  	releasedAt: Time!
```
It's seem that every change we published should require `releasedAt`. What are your thoughts on this?